src/main/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilder.java

```diff
@@ -64,7 +64,15 @@ FilterVariable generateFilter(SnapshotBuilderProgramDataRangeCriteria rangeCrite
                 new Literal(rangeCriteria.getHigh()))));
   }
 
+  FilterVariable selectAllValuesOfListCriteria() {
```

Comment:
Instead of a name specific to this use case, maybe it could have a more general name based on what it does?
```suggestion
  FilterVariable alwaysTrueFilter() {
```

Reply:
If we're going to do something like this, should we put this in a `utils` file? I can leave a comment instead of this function title.

Reply:
I think a better place would be as a `static` method in `FilterVariable`.